eHealthAfrica/move
# Men 594

#### What does this PR do?
Checks if the config data exists in the redux store as soon as the config activity loads. If yes it loads the data else queries the db before loading.

#### How to test
`react-native run-android`
Navigate to the config screen
Check the load time of data

#### Related JIRA ticket
[MEN-594](https://jira.ehealthafrica.org/browse/MEN-594)
